workspaces/ui/src/components/dashboards/TestingDashboard.js

```diff
@@ -47,13 +47,36 @@ async function ExampleReportTestingDashboardServiceBuilder(exampleId) {
   return new ExampleReportTestingDashboardService(orgId);
 }
 
-function TestingDashboardHome() {
-  debugger
-  const routingNavigationContext = useContext(RoutingNavigationContext);
-  const {baseUrl} = routingNavigationContext;
+function TestingDashboardHome(props) {
+  const { match } = props;
+  const baseUrl = match.url;
+
+  // TODO: consider factoring this as a useCaptures hook, for re-use elsewhere
+  // ApolloClient's useQuery might serve as inspiration for this: https://github.com/apollographql/apollo-client/blob/master/src/react/hooks/utils/useBaseQuery.ts
```

Comment:
Will check it out! Another alternative is to provide it via context, which might allow for greater control.